Automattic/woocommerce-payments
# Remove enqueue for `payment-request.css`

Fixes #2876

#### Changes proposed in this Pull Request

#2776 Removed `payment-request.css` but I never removed the actual "enqueue" for the file. This removes that file.

#### Testing instructions

1. Apply patch
2. Open browser console
3. Go to checkout
4. See no error for missing file /woocommerce-payments/dist/payment-request.css?ver=2.9.1

-------------------
